samples/dashboards/demo/climate/demo.css

```diff
@@ -184,6 +189,15 @@
     cursor: pointer;
 }
 
+#kpi-layout .highcharts-dashboards-component-title {
+    font-size: 1em !important;
```

Comment:
If @TorsteinHonsi added here `!important` we can assume that other users will do it in a similar fashion...
IMO we should remove that dynamic `fontSize` for KPIComponent

Reply:
Probably. My first thought was "why is this font size hard coded"? At least, an option to avoid dynamic font size would be appropriate.

Reply:
Being honest, I'm not convinced to use `!important`, I suggest to use more specific selector or inline style.

Reply:
@karolkolodziej @TorsteinHonsi, for now this is a part of discussion, so I'll just delete the `!important` flag, and we will come back to this topic in the future.